addons/udes_stock/tests/test_stock_location.py

```diff
@@ -0,0 +1,35 @@
+from .common import BaseUDES
+from odoo.exceptions import ValidationError
+
+class TestStockLocationModel(BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockLocationModel, cls).setUpClass()
+        cls.Location = cls.env['stock.location']
+
+    def test00_get_create_validate_input(self):
+        """ Check raises error if identifier is a list """
```

Comment:
Inconsistent with docstring in product.product:
* Spaces start and end
* Different docstring for same type of test